demisto_sdk/commands/content_graph/strict_objects/layouts_container.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.content_graph.strict_objects.common import (
+    DESCRIPTION_DYNAMIC_MODEL,
+    ID_DYNAMIC_MODEL,
+    NAME_DYNAMIC_MODEL,
+    BaseStrictModel,
+    create_model,
+)
+
+
+class ArgFilterSchema(BaseStrictModel):
+    operator: str
+    ignore_case: Optional[bool] = Field(None, alias="ignorecase")
+    left: dict = Field(..., example={"value": Any, "isContext": bool})
+    right: Optional[dict] = Field(None, example={"value": Any, "isContext": bool})
+    type_: Optional[str] = Field(None, alias="type")
+
+
+class ArgFiltersSchema(BaseStrictModel):
+    __root__: List[ArgFilterSchema]
```

Comment:
try a `RootModel[List[ArgFilterSchema]]` instead ([docs](https://docs.pydantic.dev/latest/concepts/models/#rootmodel-and-custom-root-types))

Reply:
It does not work for me

Reply:
Is changed to regular object with fields by the Content repo